download_data.py

```diff
@@ -64,45 +65,42 @@
 def geojson_point_feature(lat: float, lon: float, properties: Dict[str, str]) -> dict:
     return {
         "type": "Feature",
-        "geometry": {
-            "type": "Point",
-            "coordinates": [lon, lat]
-        },
+        "geometry": {"type": "Point", "coordinates": [lon, lat]},
         "properties": properties,
     }
 
 
-def elements_from_overpass_api(api_url: str, query: str) -> List[dict]:
-    logger.info(f'Requesting data from: {api_url}')
+def get_elements_from_overpass_api(api_url: str, query: str) -> List[dict]:
```

Comment:
hm z tego co do tej pory widzaiałem zazwyczaj w pythonie unika się nazywania funkcji get_costam, ale w sumie obojętnie

Reply:
A to nawet nie wiedziałem, muszę poczytać na ten temat i się ew. zmieni